app/models/product_test.rb

```diff
@@ -212,6 +212,7 @@ def pick_denom_ids
       pcv.value['medical_record_id']
     end
     denom_ids.uniq!
+    denom_ids.keep_if { |id| id[8] == '-' }
```

Comment:
I know you explain it in the PR comment, but do we want to add a comment to the code here/below to explain why we're filtering based on the presence of a `-`? It seems like future-coders may be very confused otherwise.

Reply:
Done

Reply:
👍